### PR TITLE
42 Create Users Schema

### PR DESCRIPTION
**Pull request by chris-mega**
- Replace everything related to `posts`
- Create initial User schema
- Support `.env` in backend

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
 app.use(bodyParser.json({ limit: "30mb", extended: true }));
 app.use(bodyParser.urlencoded({ limit: "30mb", extended: true }));
 app.use(cors());
 
-app.use("/posts", postRoutes); //every route inside of post routes will start with post
+app.use("/users", userRoutes); //every route inside of post routes will start with post
 
-const CONNECTION_URL =
-  "mongodb+srv://Jonathan:Jonathan123@cluster0.2rpdj.mongodb.net/<dbname>?retryWrites=true&w=majority";
-const PORT = process.env.PORT || 5000;
```
Comment (Jonathan204) [inline server/index.js:18]: does this effect everyone connecting to the same db?

**Review by Jonathan204 — APPROVED** [review]
Looks good! just had a comment about how changing the api path to an env file would effect how people deploying the app connect to the db

**Comment by Jonathan204** [comment]
nice job!

File: server/index.js (hunk 1 above)
Comment (chris-mega) [reply server/index.js:18]: We could use an account for "prod" so everyone can test ans use the same database. Using env will help if any of us want to use another key as playground

**Pull request merged**